quic/datagram.nim

```diff
@@ -0,0 +1,11 @@
+import congestion
+
+type
+  Datagram* = object
+    data*: seq[byte]
+    ecn*: ECN
+  DatagramBuffer* = openArray[byte]
```

Comment:
we tend to avoid these kind of aliases, specially with "special" types like openarray - they have a tendency to obfuscate compiler errors and code alike

Reply:
Yes, agreed. Openarray is already hard enough to understand without having an alias in between. Fixed in #16.